llvm/test/CodeGen/SPIRV/instructions/insertvalue-undef-ptr.ll

```diff
@@ -0,0 +1,27 @@
+; RUN: llc -verify-machineinstrs -O0 -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s
+
```

Comment:
Would it be possible to add a SPIR-V validator run line as well (if the validation passes)?
```suggestion
; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv64-unknown-unknown %s -o - -filetype=obj | spirv-val %}

```

Reply:
Sure. Verified locally that is passes validation.